pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.6.7</version>
+		<version>2.6.12</version>
```

Comment:
why need to update this dep?

Reply:
See comment below. It is releted to a netty error on mac-os arch 64